blockreassurance.php

```diff
@@ -125,15 +125,59 @@ public function __construct()
      */
     public function install()
     {
+        // SQL
+        $sqlQueries = [];
+        $sqlQueries[] = ' CREATE TABLE IF NOT EXISTS `' . _DB_PREFIX_ . 'psreassurance` (
+            `id_psreassurance` int(10) unsigned NOT NULL AUTO_INCREMENT,
+            `icon` varchar(255) NULL,
+            `custom_icon` varchar(255) NULL,
+            `status` int(10) unsigned NOT NULL,
+            `position` int(10) unsigned NOT NULL,
+            `id_shop` int(10) unsigned NOT NULL,
+            `type_link` int(10) unsigned NULL,
+            `id_cms` int(10) unsigned NULL,
+            `date_add` datetime NOT NULL,
+            `date_upd` datetime NULL,
+            PRIMARY KEY (`id_psreassurance`)
+        ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=UTF8;';
+        $sqlQueries[] = 'CREATE TABLE IF NOT EXISTS `' . _DB_PREFIX_ . 'psreassurance_lang` (
+            `id_psreassurance` int(10) unsigned NOT NULL,
+            `id_lang` int(10) unsigned NOT NULL,
+            `id_shop` int(10) unsigned NOT NULL,
+            `title` varchar(255) NOT NULL,
+            `description` varchar(255) NOT NULL,
+            `link` varchar(255) NOT NULL,
+            PRIMARY KEY (`id_psreassurance`,`id_shop`,`id_lang`)
+        ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=UTF8;';
+
+        $sqlInsertPSReassurance = 'INSERT INTO ' . _DB_PREFIX_ . 'psreassurance (icon, custom_icon, status, position, id_shop, type_link, id_cms, date_add) VALUES ';
+
+        $sqlQueries[] = $sqlInsertPSReassurance . "('" . $this->img_path . "reassurance/pack2/security.svg', null, 1, 1, 1, null, null, now())";
+        $sqlQueries[] = $sqlInsertPSReassurance . "('" . $this->img_path . "reassurance/pack2/carrier.svg', null, 1, 2, 1, null, null, now())";
+        $sqlQueries[] = $sqlInsertPSReassurance . "('" . $this->img_path . "reassurance/pack2/parcel.svg', null, 1, 3, 1, null, null, now())";
```

Comment:
This could be simplified in a single request. Also you forgot to escape:

```php
$sqlQueries[] = 'INSERT INTO ' . _DB_PREFIX_ . 'psreassurance (icon, custom_icon, status, position, id_shop, type_link, id_cms, date_add) VALUES
        ' . "('" . pSQL($this->img_path) . "reassurance/pack2/security.svg', null, 1, 1, 1, null, null, now()),
        ('" . pSQL($this->img_path) . "reassurance/pack2/carrier.svg', null, 1, 2, 1, null, null, now()),
        ('" . pSQL($this->img_path) . "reassurance/pack2/parcel.svg', null, 1, 3, 1, null, null, now())
    ";
```

Reply:
:ok_hand: